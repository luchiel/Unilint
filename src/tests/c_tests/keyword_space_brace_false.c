if (a)
    while(a++ < 0) {
        while (no_error) {}
    }