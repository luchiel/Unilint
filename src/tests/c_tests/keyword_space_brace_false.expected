1:3: space after if
3:14: space after while
